Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.3.json

```diff
@@ -450,7 +450,7 @@
                                             "$id": "#display/sliceProjectionUseFiducialColor",
                                             "type": "boolean",
                                             "title": "Use fiducial color for slice projection",
-                                            "description": "Choose between projetion color or fiducial color for projections.",
+                                            "description": "Choose between projection color or fiducial color for projections.",
```

Comment:
@lassoan Are there any issues updating already versioned schemas ?

Alternatively, I could exclude the files `markups-schema-v1.0.0.json` to `markups-schema-v1.0.3.json` from our codespell checks and  add an updated `markups-schema-v1.0.4.json` with the typo fixed.

Reply:
I'm not sure what the convention for this is, but I would be OK with fixing a typo in a comment without increasing version number.